### PR TITLE
Task/372 create backup seed screens

### PR DESCRIPTION
**Pull request by ciripel**

### CODE REVIEW

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart [hunk 1]
```diff
@@ -37,115 +40,173 @@ class _BackupInventoryScreen extends State<BackupInventoryScreen> {
...
+                    Align(
+                      alignment: Alignment.center,
+                      child: Text(
+                        context.locale.backup_inventory_screen_your_secret,
+                        style: SioTextStyles.h3.apply(
+                          color: SioColors.whiteBlue,
+                        ),
+                      ),
+                    ),
+                    Gaps.gap4,
```
Comment (JiriBestak) [inline lib/view/screens/authenticated_screens/backup_inventory_screen.dart:88]: We talked about this Gaps with Filip and its not default thing of Flutter. We should use some default component of flutter. SizedBox(height: 10) for Example

File: lib/view/widgets/welcome_screen_page.dart [hunk 2]
```diff
@@ -36,12 +36,10 @@ class WelcomeScreenPage extends StatelessWidget {
                     ),
                   ),
                   Gaps.gap5,
-                  Text(
-                    subtitle,
-                    style: SioTextStyles.s1.apply(
-                      color: SioColorsDark.secondary7,
-                    ),
-                  ),
+                  Text(subtitle,
```
[the remark this answers is not in the document]
Comment (JiriVrbas-Eria) [reply lib/view/widgets/welcome_screen_page.dart:39]: There are everywhere

File: lib/view/screens/authenticated_screens/backup_inventory_screen.dart (hunk 1 above)
Comment (JiriVrbas-Eria) [reply lib/view/screens/authenticated_screens/backup_inventory_screen.dart:87]: Now sure how to use it properly rn, will skip it

File: lib/view/screens/authenticated_screens/backup_inventory_tutorial_screen.dart [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return SioScaffold(
+      body: SafeArea(
+        child: Stack(
+          children: [
+            Positioned.fill(
+              child: Story(
+                repeat: !const bool.fromEnvironment('TEST_RUN'),
```
[the remark this answers is not in the document]
Comment (JiriVrbas-Eria) [reply lib/view/screens/authenticated_screens/backup_inventory_tutorial_screen.dart]: Ok, removed

**Review by storkandstars — APPROVED** [review]

**Pull request merged**